test-clients/src/main/resource/contract/bytecodes/LargeContract.bin

```diff
@@ -0,0 +1 @@
c65a03f11515614d8057600080fd5b505050604051805190509150816fffffffffffffffffffffffffffffffff1682141515614dac57600080fd5b600282811515614db857fe5b0482019050662386f26fc10000811015614dd757662386f26fc1000090505b809250505090565b6000808260a0015163ffffffff16148015614e1257504367ffffffffffffffff16826040015167ffffffffffffffff1611155b9050919050565b6000806000600685815481101515614e2d57fe5b90600052602060002090600202019150600684815481101515614e4c57fe5b90600052602060002090600202019050614e6882868387614686565b9250505092915050565b600080600683815481101515614e8457fe5b90600052602060002090600202019150600684815481101515614ea357fe5b90600052602060002090600202019050828160010160186101000a81548163ffffffff021916908363ffffffff160217905550614edf82615087565b614ee881615087565b600a600085815260200190815260200160002060006101000a81549073ffffffffffffffffffffffffffffffffffffffff0219169055600a600084815260200190815260200160002060006101000a81549073ffffffffffffffffffffffffffffffffffffffff0219169055600f600081548092919060010191905055507f241ea03ca20251805084d27d4440371c34a0b85ff108f6bb5611248f73818b806007600086815260200190815260200160002060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1685858460010160089054906101000a900467ffffffffffffffff16604051808573ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff1681526020018481526020018381526020018267ffffffffffffffff16815260200194505050505060405180910390a150505050565b60005b602082101515615064578251845260208401935060208301925060208203915061503f565b6001826020036101000a0390508019835116818551168181178652505050505050565b43600554600383600101601c9054906101000a900461ffff1661ffff16600e811015156150b057fe5b600891828204019190066004029054906101000a900463ffffffff1663ffffffff168115156150db57fe5b04018160010160086101000a81548167ffffffffffffffff021916908367ffffffffffffffff160217905550600d81600101601c9054906101000a900461ffff1661ffff16101561515a57600181600101601c8282829054906101000a900461ffff160192506101000a81548161ffff021916908361ffff1602179055505b50565b602060405190810160405280600081525090565b6080604051908101604052806004905b60008019168152602001906001900390816151815790505090565b602060405190810160405280600081525090565b6101006040519081016040528060008152602001600067ffffffffffffffff168152602001600067ffffffffffffffff168152602001600063ffffffff168152602001600063ffffffff168152602001600063ffffffff168152602001600061ffff168152602001600061ffff1681525090565b815481835581811511615251576002028160020283600052602060002091820191016152509190615256565b5b505050565b61531f91905b8082111561531b576000808201600090556001820160006101000a81549067ffffffffffffffff02191690556001820160086101000a81549067ffffffffffffffff02191690556001820160106101000a81549063ffffffff02191690556001820160146101000a81549063ffffffff02191690556001820160186101000a81549063ffffffff021916905560018201601c6101000a81549061ffff021916905560018201601e6101000a81549061ffff02191690555060020161525c565b5090565b905600a165627a7a72305820b3c51888dee029283a3354778045a96d67164bf7b6ec1254d527e0ac3fdf3c1c0029
```

Comment:
Lets rename this to CryptoKitties.sol. This way we will **know** the solidity mapping to the binary